element_calcium_imaging/imaging_no_curation.py

```diff
@@ -215,6 +215,8 @@ def auto_generate_entries(cls, scan_key, task_mode):
                 "task_mode": task_mode,
             }
         )
+    
+    generate = auto_generate_entries
```

Comment:
If `generate` going to become the main name, then I would rename the function and provide an alias for backward compatibility.

Reply:
Updated.  Thanks.